src/viewer.py

```diff
@@ -530,6 +530,7 @@ def __init__(self):
         # Create tree view widget
         self.tree = wx.TreeCtrl(nb,-1,style=wx.TR_HAS_BUTTONS)
         self.tree.Bind(wx.EVT_TREE_SEL_CHANGED, self.OnTreeSelectionChanged)
+        self.tree.Bind(wx.EVT_TREE_ITEM_ACTIVATED, self.on_trigger_transition)
```

Comment:
Either stick to the existing naming convention (_CamelCase_) or convert all method names to _lower_case_.

Reply:
That method already exists though, I did not add it. Upper camel case is used less than snake case so if you want, I can rename all existing camel case methods to snake case. I don't think it's a fit for this PR though.

Reply:
Yes, please change to snake_case. I'm fine if you do it in a separate PR.

Reply:
After some thought, due to wxPython having – as you mentioned – upper camel case as its naming convention, maybe we should convert everything to upper camel case. On the other hand, if all other methods are snake case, wx methods stand out nicely.